tendrl/monitoring_integration/webhook/webhook_receiver.py

```diff
@@ -1,85 +1,69 @@
+            alert_handler = AlertHandlerManager()
+            data = request.json
+            if "ruleId" in data:
+                alert_handler.handle_alert(
+                    data["ruleId"]
+                )
             else:
-                data = env['wsgi.input'].read(
-                    int(env['CONTENT_LENGTH'])
+                logger.log(
+                    "debug",
+                    NS.publisher_id,
+                    {
+                        "message": "Unable to find ruleId %s" % data
+                    }
                 )
-                data = json.loads(data)
-                if "ruleId" in data:
-                    self.alert_handler.handle_alert(
-                        data["ruleId"]
-                    )
-                    response = Response('Alert received successfully')
-                    response.headers['content-length'] = len(response.data)
-                    response.status_code = 200
-                else:
-                    logger.log(
-                        "debug",
-                        NS.publisher_id,
-                        {
-                            "message": "Unable to find ruleId %s" % data
-                        }
-                    )
         except (IOError, AssertionError, KeyError) as ex:
             Event(
                 ExceptionMessage(
                     priority="debug",
                     publisher=NS.publisher_id,
                     payload={
-                        "message": "Unable to read alert from socket",
+                        "message": "Unable to handle grafana alert",
                         "exception": ex
                     }
                 )
             )
-            response = Response('Error in reading alert from socket')
-            response.headers['content-length'] = len(response.data)
-            response.status_code = 500
-        return response(env, start_response)
+        return "OK"
 
     def run(self):
-        try:
-            run_simple(
-                socket.gethostbyname(socket.gethostname()),
-                PORT,
-                self._application, threaded=True
-            )
-        except (TypeError,
-                ValueError) as ex:
-            Event(
-                ExceptionMessage(
-                    priority="debug",
-                    publisher=NS.publisher_id,
-                    payload={
-                        "message": "Unable to start wehook receiver",
-                        "exception": ex
-                    }
-                )
-            )
+        app_logged = TransLogger(app)
+        cherrypy.tree.graft(app_logged, '/')
+        cherrypy.config.update({
+            'engine.autoreload_on': False,
+            'log.screen': True,
+            'server.socket_port': PORT,
+            'server.socket_host': socket.gethostbyname(
+                socket.gethostname()),
+            'log.access_file': '',
+            'log.error_file': ''
```

Comment:
Shouldnt we use a proper error file to log any errors here?

Reply:
so we are making cherrypy log to stdout/stderr at L#57

This will ensure those logs are logged as per Tendrl's logger

@GowthamShanmugam please verify this on your setup